### PR TITLE
Move parseTemplate for log probes to tests part

### PR DESCRIPTION
**Pull request by jpbempel**
only used for tests

# What Does This Do

# Motivation
code cleanup

# Additional Notes

### CODE REVIEW

File: dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/probe/LogProbeTest.java [hunk 1]
```diff
@@ -52,7 +53,10 @@ public void log() {
     assertEquals("}", logProbe.getSegments().get(5).getStr());
   }
 
-  private LogProbe.Builder createLog() {
-    return LogProbe.builder().language(LANGUAGE).probeId(PROBE_ID);
+  private LogProbe.Builder createLog(String template) {
+    return LogProbe.builder()
+        .language(LANGUAGE)
+        .probeId(PROBE_ID)
+        .template(template, parseTemplate(template));
```
Comment (shatzi) [inline dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/probe/LogProbeTest.java:60]: nit: add build() at the end, as it always been used the same.

File: dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/agent/LogMessageTemplateSummaryBuilderTest.java [hunk 2]
```diff
@@ -219,4 +220,8 @@ public void argInaccessibleFieldTemplate() {
       assertTrue(evaluationErrors.get(i).getMessage().contains("Cannot extract field:"));
     }
   }
+
+  private LogProbe createLogProbe(String template) {
```
Comment (shatzi) [inline dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/agent/LogMessageTemplateSummaryBuilderTest.java:224]: nit: maybe we can move this to LogProbeTestHelper createLogProbeFromTemplate(template) - as this been used in multiple places?

**Review by shatzi — APPROVED** [review]
Nice cleanup

File: dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/agent/LogMessageTemplateSummaryBuilderTest.java (hunk 2 above)
Comment (jpbempel) [reply dd-java-agent/agent-debugger/src/test/java/com/datadog/debugger/agent/LogMessageTemplateSummaryBuilderTest.java:224]: not sure it is worth the effort, there are subtleties each time, I prefer to keep private helper method in each test class

**Pull request merged**